setup-akash-console.sh

```
@@ -0,0 +1,440 @@
+
+echo "Checking required software..."
+
+# Check Node.js
+if command -v node &> /dev/null; then
+    NODE_VERSION=$(node --version | sed 's/v//')
+    show_success "Node.js found: v$NODE_VERSION"
+
+    # Check if version is compatible (major version should be 22+, minor 14+)
+    NODE_MAJOR=$(echo $NODE_VERSION | cut -d. -f1)
+    NODE_MINOR=$(echo $NODE_VERSION | cut -d. -f2)
+    NODE_PATCH=$(echo $NODE_VERSION | cut -d. -f3)
```

Shellcheck (0.10.0)

[warning] NODE_PATCH appears unused. Verify use (or export if used externally).

(SC2034)